pkg/local_object_storage/pilorama/interface.go

```diff
@@ -37,6 +37,9 @@ type Forest interface {
 	// TreeDrop drops a tree from the database.
 	// If the tree is not found, ErrTreeNotFound should be returned.
 	TreeDrop(cid cidSDK.ID, treeID string) error
+	// TreeGetTrees returns all the tree IDs that have been added to the
+	// passed container ID. Nil slice should be returned if no tree found.
+	TreeGetTrees(cid cidSDK.ID) ([]string, error)
```

Comment:
Can we build `neofs-node` on this commit?

Reply:
i was surprised how "useful" `Forest` intreface is (defined in tree storage pkg, implemented there too, used for tests and is forced to be implemented by the `engine` (sub)pkgs). so the answer is "no" cause i did not want to make all changes in a one huge commit and im always think that small atomic commits > being possible to perform correct building on every commit. but ok, ive added dummy implementation in the engine here, and now every commit could be build

Reply:
One could argue that if we cannot build after a commit, is it really _atomic_? Building on any commit is a must-have, for example, because `git bisect` can accidentally checkout on it..